src/main/cpp/t08_chess.cpp

```diff
@@ -19,11 +19,39 @@
 
 #include "t08_chess.h"
 #include <iostream>
+#include <iomanip>
 
 
 using namespace std;
 
 int t08_chess() {
-    
-    return 0;
+	int n, m, l = 1;
+	cin >> n >> m;
+	int mas[n][m];
+	for (int i = 0; i < n; i++) {
+		for (int j = 0; j < m; j++) {
+			if ((i + j) % 2 == 0)
+			{
+				mas[i][j] = l;
+				l++;
+			}
+			else if (i + j == 0) {
+				mas[i][j] = l;
+				l++;
+			}
+			else if ((i + j) % 2 != 0) mas[i][j] = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Condition '(i+j)%2!=0' is always true](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4464668)